README.md

```diff
@@ -19,10 +19,10 @@ You need Python 3 and tox. Install tox with pip or use the package manager of yo
 
 ## Building the documentation
 
-Run ``tox -e docs`` that will run sphinx-build and the built documentation
+Run ``tox -e docs`` to run sphinx-build. The built documentation
 will be in the ``build/html`` directory.
 
 ## Build and preview documentation locally
 
-Run ``tox -e run`` that will build the documentation and serve it locally
+Run ``tox run -e run`` to build the documentation and serve it locally
```

Comment:
This looks wrong ``tox -e run`` should work, what issues do you get?

```
tobias.urdin@MacBook-Pro-TU documentation % (main) tox --version
3.27.1 imported from /Users/tobias.urdin/Library/Python/3.9/lib/python/site-packages/tox/__init__.py
tobias.urdin@MacBook-Pro-TU documentation % (main) tox -e run   
run create: /Users/tobias.urdin/src/github.com/tobias-urdin/documentation/.tox/run
run installdeps: -r/Users/tobias.urdin/src/github.com/tobias-urdin/documentation/requirements.txt
run installed: alabaster==0.7.13,annotated-types==0.5.0,appdirs==1.4.4,Babel==2.12.1,certifi==2023.7.22,charset-normalizer==3.2.0,click==8.1.7,docutils==0.20.1,flake8==6.1.0,idna==3.4,imagesize==1.4.1,importlib-metadata==6.8.0,Jinja2==3.1.2,MarkupSafe==2.1.3,mccabe==0.7.0,packaging==23.1,pycodestyle==2.11.0,pydantic==2.3.0,pydantic-extra-types==2.1.0,pydantic_core==2.6.3,pyflakes==3.1.0,Pygments==2.16.1,requests==2.31.0,snowballstemmer==2.2.0,Sphinx==7.2.5,sphinx-immaterial==0.11.7,sphinx-intl==2.1.0,sphinx-sitemap==2.5.1,sphinxcontrib-applehelp==1.0.7,sphinxcontrib-devhelp==1.0.5,sphinxcontrib-htmlhelp==2.0.4,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.6,sphinxcontrib-serializinghtml==1.1.9,typing_extensions==4.7.1,urllib3==2.0.4,zipp==3.16.2
run run-test-pre: PYTHONHASHSEED='2784940760'
run run-test: commands[0] | sphinx-build -E -W -d build/doctrees -b html . build/html
Running Sphinx v7.2.5
/Users/tobias.urdin/src/github.com/tobias-urdin/documentation/.tox/run/lib/python3.9/site-packages/urllib3/__init__.py:34: NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 154 source files that are out of date
updating environment: [new config] 154 added, 0 changed, 0 removed
reading sources... [100%] storage/storage-types
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
copying assets... copying static files... done
copying extra files... done
done
writing output... [100%] storage/storage-types
generating indices... genindex done
writing additional pages... done
dumping search index in English (code: en)... done
dumping object inventory... done
sphinx-sitemap: sitemap.xml was generated for URL https://docs.binero.com/ in /Users/tobias.urdin/src/github.com/tobias-urdin/documentation/build/html/sitemap.xml
build succeeded.

The HTML pages are in build/html.
run run-test: commands[1] | python3 -m http.server -d build/html
Serving HTTP on :: port 8000 (http://[::]:8000/) ...
^C
Keyboard interrupt received, exiting.
ERROR: got KeyboardInterrupt signal
___________________________________________________________________________________________________________________________________ summary ____________________________________________________________________________________________________________________________________
ERROR:   run: keyboardinterrupt
``

Reply:
It felt very odd, but that was the only way I could get it to work. Here is the output.
```
max@petrichor ~/repos/documentation [main]
> tox -e run
usage: tox run [-h] [--colored {yes,no}] [--exit-and-dump-after seconds] [-c file] [--workdir dir] [--root dir] [--runner {virtualenv}] [-v | -q] [-e ENV | -m label [label ...] | -f factor [factor ...] | --skip-env re] [--result-json path]
               [-s [v]] [-n] [-b] [--installpkg INSTALL_PKG] [--develop] [--hashseed SEED] [--discover path [path ...]] [--no-recreate-pkg] [--list-dependencies | --no-list-dependencies] [--skip-pkg-install] [--version]
               [--no-provision [REQ_JSON]] [--no-recreate-provision] [-r] [-x OVERRIDE]
tox run: error: argument -e: expected one argument
```

Reply:
Maybe I lack some configuration or an alias for tox?

Reply:
What version of tox is that?

```
tobias.urdin@MacBook-Pro-TU ~ % which tox
/Users/tobias.urdin/Library/Python/3.9/bin/tox
tobias.urdin@MacBook-Pro-TU ~ % tox --version
3.27.1 imported from /Users/tobias.urdin/Library/Python/3.9/lib/python/site-packages/tox/__init__.py
tobias.urdin@MacBook-Pro-TU ~ % alias | grep tox
tobias.urdin@MacBook-Pro-TU ~ % 
```

Reply:
I assume you are using version 4 then right? https://tox.wiki/en/latest/upgrading.html#updating-usage-with-e

Reply:
4.11.1

Reply:
You can change `tox -e` commands to `tox run -e`` and update minversion in ``tox.ini`` file to 3.27 and thus both major versions should work with that.

Reply:
Couldn't get that to work easily, aliases or functions can't have a space in the name so I just downgraded instead. Could perhaps write that you need tox 3 or earlier?

Reply:
Sure, you can update documentation to say tox 3. I can fix a follow-up patch later adding tox 4 support.